src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.objects.Ball;
+import java.util.Random;
+
+public class Lottery {
+    private final Random random;
+    private final ColorSupplier colorSupplier;
+
+    public Lottery() {
+        // SHOULD I MAKE RANDOM AS CLASS FIELD OR JUST AS SIMPLE LOCAL FUNCTION VARIABLE ?
+        this.random = new Random();
+        colorSupplier = new ColorSupplier();
+    }
+
+    public Ball getRandomBall(int maxNum) {
+        // SHOULD maxNum be included in the range or not ?
```

Comment:
according to the task:
"the maximum possible number should be 100"
so yes, it should be included.